src/time_travel/patchers/basic_patcher.py

```diff
@@ -10,6 +10,11 @@ def __init__(self, clock, events_pool=None):
         self.events_pool = events_pool
         
         self.patches = []
+        
+    @classmethod
+    def register_events_types(cls, main_events_types):
```

Comment:
It's the virtual patcher. It doesn't any more, but it has the new interface to register patchers.